caab-service/src/main/java/uk/gov/laa/ccms/caab/api/service/ApplicationService.java

```diff
@@ -69,6 +71,100 @@ public ApplicationDetail getApplication(final Long applicationId) {
             HttpStatus.NOT_FOUND));
   }
 
+  /**
+   * Gets an application's linked cases.
+   *
+   * @param id the TDS id for the application.
+   * @return the application's linked cases.
+   */
+  @Transactional
+  public List<LinkedCase> getLinkedCasesForApplication(final Long id) {
+    return applicationRepository.findById(id)
+        .map(Application::getLinkedCases)
+        .map(applicationMapper::toLinkedCaseModelList)
+        .orElseThrow(() -> new CaabApiException(
+            String.format("Linked cases for application with id %s not found", id),
+            HttpStatus.NOT_FOUND));
+  }
+
+  /**
+   * Creates and associates a new linked case with a specified application.
+   * If the application is not found, a CaabApiException is thrown.
+   *
+   * @param id The unique identifier of the application.
+   * @param linkedCase The LinkedCase object containing the details of the case to be linked.
+   * @throws CaabApiException If the application with the specified ID is not found.
+   */
+  @Transactional
+  public void createLinkedCaseForApplication(final Long id, final LinkedCase linkedCase) {
```

Comment:
Change param name to `applicationId` for ease of understanding

Reply:
done